dev_app.ps1

```diff
@@ -4,26 +4,30 @@
 
 Disable-UAC
 
-#--- Windows Features ---
-Set-WindowsExplorerOptions -EnableShowHiddenFilesFoldersDrives -EnableShowProtectedOSFiles -EnableShowFileExtensions
+invoke-expression -ConnectionUri https://raw.githubusercontent.com/crutkas/windows-dev-box-setup-scripts/master/helper_WindowsExplorerSettings.ps1

```

Comment:
will need to change crutkas to windows.  

When I run it I get "ERROR: A parameter cannot be found that matches parameter name 'ConnectionUri'."